abis/linux/statx.h

```diff
@@ -0,0 +1,38 @@
+#ifndef _ABIBITS_STATX_H
+#define _ABIBITS_STATX_H
+
+#include <bits/types.h>
+
+struct statx_timestamp {
+	__mlibc_int64 tv_sec;
+	__mlibc_uint32 tv_nsec;
+	__mlibc_uint32 __padding;
+};
+
+struct statx {
```

Comment:
Given that linux documents that the size of this struct is `0x100`, we should static_assert on it.